internal/integration/tests/rethinkdb_test.go

```diff
@@ -53,16 +51,14 @@ func init() {
 	}
 }
 
-//
 // Begin TestMain(), Setup, Teardown
-//
 func testSetup(m *testing.M) {
 	var err error
 	session, err = r.Connect(r.ConnectOpts{
 		Address: url,
 	})
 	if err != nil {
-		r.Log.Fatalln(err.Error())
+		panic(err)
```

Comment:
Not sure about the rest of the file, but would be good to follow the same pattern as previously, like something similar to:

```suggestion
		panic(fmt.Errorf("Error connecting to instance: %w", err))
```

Reply:
done